src/ProjectTemplates/Web.ProjectTemplates/content/BlazorWeb-CSharp/BlazorWeb-CSharp/wwwroot/css/app.css

```diff
@@ -1,3 +1,6 @@
+@import url("bootstrap/bootstrap.min.css");
```

Comment:
I'm not sure if moving these CSS imports here is simplifying or obfuscating for new users :thinking:. Are new users more likely to be familiar with link tags in an HTML doc or CSS imports? My gut says they will be more familiar with the former.

Reply:
I don't mind changing it back if you prefer. It would mean having three CSS link tags in the top-level HTML instead of one.

It's pretty common to structure things this way with `@import` as it gives more separation of concerns, but having separate top-level tags helps with parallelising the page download.

Altogether I don't mind much.